### PR TITLE
Card Script for Blur

### LINKED ISSUE #683: Blur

### PR DESCRIPTION
**Pull request by mctubbies**
closes #683

### CODE REVIEW

File: Blur.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Blur
+ManaCost:2 U
+Types:Instant
+A:SP$ ChangeZone | Cost$ 2 U | ValidTgts$ Creature.YouCtrl | TargetMin$ 1 | TargetMax$ 1 | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature you control | RememberTargets$ True | SubAbility$ DBReturn | SpellDescription$ Exile target creature you control, then return that card to the battlefield under their owner's control.
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ All | Destination$ Battlefield | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ 1 | SpellDescription$ Draw a card.
+AI:RemoveDeck:All
```
Comment (Northmoc) [inline Blur.txt:8]: Hmm, seems like the AI could handle this... why did you add this? Is there another blink type spell with this tag?

**Review by Northmoc — COMMENTED** [review]
Great!

See what you think about that AI question

File: Blur.txt (hunk 1 above)
Comment (mctubbies) [reply Blur.txt]: I cannot remember which card I referenced that had it. Looking at "Acrobatic Maneuver" (which is the same card but white) it does not have that line so I'll just remove it.

**Review by Northmoc — APPROVED** [review]
Looks good to me!

**Comment by paulsnoops** [comment]
Please change the card script file name to: blur.txt
and move to: \forge\forge-gui\res\cardsfolder\upcoming

File: Blur.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Blur
+ManaCost:2 U
+Types:Instant
+A:SP$ ChangeZone || ValidTgts$ Creature.YouCtrl | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature you control | RememberTargets$ True | SubAbility$ DBReturn | SpellDescription$ Exile target creature you control, then return that card to the battlefield under their owner's control.
```
Comment (tool4ever) [inline Blur.txt:4]: ||

**Review by tool4ever — APPROVED** [review]
